app/experimenter/nimbus-ui/src/components/App/index.tsx

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from "react";
-import AppLayout from "./AppLayout";
+import AppLayout from "../AppLayout";
```

Comment:
Oh you don't do import path aliases so it always looks like `"/nimbus-ui/Applayout"` using absolute imports instead of relative ones?

Reply:
@jaredlockhart I did notice when going through `rapid` that you do absolute imports, on FxA we've done relative - I like relative since it's shorter (and in tests we can `import MyComponent from "."` - definitely up for hearing reasoning for preferring absolute :)

Reply:
I'm cool with either way, so long as we're consistent across the whole of nimbus-ui. I can make this change here if we'd like.

Reply:
Oh yeah if it's not a pattern you typically prefer no need to block on it.  I just like absolute because it makes it easier to search for stuff when you're looking for every reference to something, especially if you move stuff around, but TS is also better about keep track of that so maybe it's a moot point.  Anyways don't change it on my account, I was just curious :)